Geovalidation.md

```diff
@@ -94,7 +97,7 @@ Properties of the **validation_data** object will contain the following properti
 |   |   |
 |---|---|
 | status | The status of the valdation strategy can be (NOT_RUN, SUCCESS, FAILURE, NO_RESULTS) **This property is always present and non-empty.** |
-| message | The unabbreviated form of the administrative area's name. **This property is not always present.** |
+| message | A short message describing a reason for the returned status. **This property is not always present.** |
```

Comment:
could we give more info on that last part? is it only present for non-success responses, or what?